app/services/allocation_migration_service.rb

```diff
@@ -0,0 +1,45 @@
+require "csv"
+
+class AllocationMigrationService
+  def execute
+    Allocation.all.each do |allocation|
+      accredited_body = allocation.accredited_body
+      provider = allocation.provider
+
+      allocation.accredited_body_code = accredited_body.provider_code
+      allocation.provider_code = provider.provider_code
+
+      previous_accredited_body = previous_recruitment_cycle.providers.find_by!(provider_code: accredited_body.provider_code)
+
+      if previous_accredited_body.accrediting_provider == "not_an_accredited_body"
+        previous_accredited_body.update(accrediting_provider: "accredited_body")
```

Comment:
https://github.com/DFE-Digital/teacher-training-api/blob/10782c7cdf1cdd24c8fc2bf717b05a2027fb334f/app/models/provider.rb#L240-L247

what is the `provider_type`?

Reply:
the ones being change in the migration are all `scitt`s